src/angular/components/add-edit.component.ts

```diff
@@ -81,6 +83,7 @@ export class AddEditComponent implements OnInit {
     uriMatchOptions: any[];
     ownershipOptions: any[] = [];
     currentDate = new Date();
+    allowPersonal: boolean = true;
```

Comment:
you dont have to specify type if you init it inline like this

Reply:
Resolved.